website/docs/reference/administrative-commands/createIndexes.mdx

```diff
@@ -0,0 +1,165 @@
+| `indexes`                 | array    | An array of index specification documents. Each document defines an index.                                                                      |
+| `key`                     | document | A single document that contains the field(s) to index and their sort order or type (`1`, `-1`, `text`, `cosmosSearch` for vector indexes, etc). |
+| `name`                    | string   | A required name for the index. Must be unique to the collection.                                                                                |
+| `unique`                  | boolean  | Ensures all values are unique for the indexed field if set to `true`.                                                                           |
+| `partialFilterExpression` | document | Restricts index to documents matching the condition.                                                                                            |
+| `sparse`                  | boolean  | Only indexes documents that contain the indexed field if set to `true`.                                                                         |
+
+## Example usage
+
+The following examples demonstrate how to create different types of indexes.
+
+### Simple index
+
+The simplest way to create an index is to use the `createIndexes` command with the `key` field (the field(s) to be indexed) and their sort order (`1` for ascending, `-1` for descending).
+
+This example creates a simple index on the `title` field in the `books` collection.
+
+<CodeBlock language="js">{CreateIndex}</CodeBlock>
+
+### Compound index
+
+Compound indexes enable multiple fields to be indexed together.
+
+The example below creates a compound index on the `publication.date` and `analytics.average_rating` fields in the `books` collection.
+
+<CodeBlock language="js">{CreateCompoundIndex}</CodeBlock>
+
+### Unique index
+
+Unique indexes ensure that your indexed fields only contain unique values, without duplicates.
+
+```js
+db.runCommand({
+  createIndexes: '<collection_name>',
+  indexes: [
+    {
+      key: {
+        '<field>': 1
+      },
+      name: '<unique_index_name>',
+      unique: true
+    }
+  ]
+})
+```
+
+This example creates a unique index on the `isbn` field in the `books` collection to ensure that each book has a unique ISBN.
+
+<CodeBlock language="js">{CreateUniqueIndex}</CodeBlock>
+
+### Partial index
+
+A partial index is similar to a regular index, however, it only indexes a fraction of documents in a collection based on a specified filter condition.
+
+Let's create a partial index on the `books` collection only for books published after the year 2000.
+
+<CodeBlock language="js">{CreatePartialIndex}</CodeBlock>
+
+### Sparse index
+
+Sparse indexes only index documents with the specified field, even if the field is not present in all documents.
```

Comment:
Here I think you want to highlight the fact that the documents without the specified field will not be indexed. I think following usage of words doesn't quite get the message across :thinking: 

> even if the field is not present in all documents.